src/cli.rs

```diff
@@ -305,9 +413,48 @@ pub fn make_cli_subcommands<'a, 'b>() -> App<'a, 'b> {
                         .number_of_values(1)
                         .required(true)
                         .multiple(true))
-                )
+        )
 }
 
+/// Add [clap](https://docs.rs/clap/) global wallet options to an `App`
+///
+/// Additional app specific options can be added. See [clap](https://docs.rs/clap/) for more
+/// information on how to customize the global flags for an `App`.
+///
+/// # Wallet global flags
+///
+/// | Short | Long  | Description |
+/// |-------|-------|-------------|
+/// | -c | --change_descriptor <DESCRIPTOR>            | Sets the descriptor to use for internal addresses |
+/// | -d | --descriptor <DESCRIPTOR>                   | Sets the descriptor to use for the external addresses |
+/// | -e | --esplora <ESPLORA>                         | Use the esplora server if given as parameter |
+/// |    | --esplora_concurrency <ESPLORA_CONCURRENCY> | Concurrency of requests made to the esplora server [default: 4] |
```

Comment:
We may want to avoid square brackets if not used for rust links?
It looks it's working correctly if doesn't match a rust type however I am not sure it's a good idea

Reply:
I've switched to StructOpt and removed this from docs.